### PR TITLE
[DEP][Spec] Add more examples to introduce the APIs usage

### PR DESCRIPTION
**Pull request by qjia7**
This PR will add three use cases:
1. Load a depth photo from file system
2. Load a depth photo from network
3. Refocus a depth photo

### CODE REVIEW

**Comment by qjia7** [comment]
@huningxin Please take a look.

File: spec/depth-enabled-photography.html [hunk 1]
```diff
@@ -2194,6 +2194,86 @@
   console.log('Got a photo');
 }
       </pre>
+      <h3>
+        Load a Depth Photo from Native
```
Comment (huningxin) [inline spec/depth-enabled-photography.html]: Just "Load a Depth Photo from File System" would be fine.

File: spec/depth-enabled-photography.html [hunk 2]
```diff
@@ -2194,6 +2194,86 @@
...
+          return;
+        }
+
+        // photo is the instance of DepthPhoto
+        // which can be got from the camera [Example 1] or loaded from somewhere [Example 2 or 3]
+        refocus.init(photo).then(
+            function() {
+              // [x, y] is the refocus point that you set
+              refocus.apply({ x: x, y: y }, 50.0).then(
+                  function(refocusPhoto) {
```
Comment (huningxin) [inline spec/depth-enabled-photography.html]: variable name "refocusedPhoto".

File: spec/depth-enabled-photography.html [hunk 3]
```diff
@@ -2194,6 +2194,86 @@
...
+          console.log(e.message);
+          return;
+        }
+
+        // photo is the instance of DepthPhoto
+        // which can be got from the camera [Example 1] or loaded from somewhere [Example 2 or 3]
+        refocus.init(photo).then(
+            function() {
+              // [x, y] is the refocus point that you set
+              refocus.apply({ x: x, y: y }, 50.0).then(
```
Comment (huningxin) [inline spec/depth-enabled-photography.html]: Add one comment for `aperture` parameter, as default value is 50.0. You may remove it for simplicity.

**Comment by qjia7** [comment]
@huningxin Have solved all the comments. Please take another look. Thanks.

**Comment by huningxin** [comment]
LGTM. Merging.

**Pull request merged**